### PR TITLE
Add RV as a participant using the new Live Review schema

### PR DESCRIPTION
**Pull request by eloisebrosseau**
### Summarize your change.

The `read_otio_string` function was added to parse the OTIO object containing the timeline sent by the Review App when RV is joining a new session as a participant. Since the timeline contains some annotation fields, the annotation schema was added in order to let OTIO know that it is a known schema during the deserialization step. However, nothing is done with the annotations at this stage.

### Describe the reason for the change.

This is the first step in adding back the new OTIO schema as the preferred way of communication between RV and the Review App.

### Describe what you have tested and on which operating system.

Live review using the new schema was tested between a local Review App where the feature was enabled with https://creative-collab.dev.shotguncloud.com and RV on MacOS.

### CODE REVIEW

File: src/plugins/rv-packages/otio_reader/otio_reader.py [hunk 1]
```diff
@@ -59,6 +59,24 @@ class NoNodeFromHook(otio.exceptions.OTIOError):
...
+    """
+    Main entry point to expand a given otio string into the current RV session.
+
+    Returns the top level node created that represents this otio
+    timeline.
+    """
+    otio_obj = otio.adapters.read_from_string(otio_string)
+    timeline = otio_obj["otio"]
+
+    if host_prefix is not None:
```
Comment (rogernelson) [inline src/plugins/rv-packages/otio_reader/otio_reader.py:72]: Simpler way of doing that:
```
context = {"sg_url": host_prefix} if host_prefix else None
```

File: src/plugins/rv-packages/otio_reader/otio_reader.py [hunk 2]
```diff
@@ -276,12 +294,12 @@ def _create_track(in_seq, context=None):
     return new_seq
 
 
-def _get_global_transform(tl):
+def _get_global_transform(tl) -> dict:
```
Comment (rogernelson) [inline src/plugins/rv-packages/otio_reader/otio_reader.py:297]: Great, thanks for adding these.

**Review by rogernelson — APPROVED** [review]
Looks good, thanks!

**Review by bernie-laberge — APPROVED** [review]
LGTM
Thank you Éloïse!

**Pull request merged**